public/server/db.json

```
@@ -346,6 +346,13 @@
   
   "fgs": {
     "definition": "For God's sake"
-  }
+  },
+
+  "tbh": {
+    "definition": "To be Honest"
+  },
   
+  "asap": {
+    "definition": "As soon As Possible"
```

Thank you SO much for these additions. Perhaps you could change the first letter of the word 'soon' to uppercase, just as I have suggested below
```suggestion
    "definition": "As Soon As Possible"
```

---

alright